widgetry/src/widgets/fan_chart.rs

```diff
@@ -164,7 +166,12 @@ impl FanChart {
         let mut col = Vec::new();
         for i in 0..num_y_labels {
             let percent_y = (i as f64) / ((num_y_labels - 1) as f64);
-            col.push(max_y.from_percent(percent_y).prettyprint().text_widget(ctx));
+            col.push(
+                max_y
+                    .from_percent(percent_y)
+                    .prettyprint(&unit_fmt)
```

Comment:
Because `UnitFmt` implements `Copy`, we could omit the `&` here and elsewhere. I checked with `cargo clippy` and it didn't complain, so not a huge deal in this case. I'm fine with how this is. If `UnitFmt` ever grows one day beyond 2 bools (seems unlikely), this is slightly more future proof.